GitSearch/Application/Search/SearchController.swift

```diff
@@ -10,15 +10,17 @@ import UIKit
 
 class SearchController: UIViewController, UITextFieldDelegate {
     
-    var shareView: SearchControllerView!
+    var childView: SearchControllerView!
 
     override func viewDidLoad() {
         super.viewDidLoad()
     
-        shareView = SearchControllerView(frame: self.view.bounds)
-        self.view.addSubview(shareView)
-        shareView.setNeedsUpdateConstraints()
-        
+        childView = SearchControllerView(frame: self.view.bounds)
+        self.view.addSubview(childView)
+        childView.setNeedsUpdateConstraints()
+        childView.actionButton.addTarget(self, action: #selector(buttonAction(_:)), for: .touchUpInside)
+        childView.inputText.delegate = self as UITextFieldDelegate
```

Comment:
I little worrying about this thing in superview, but for me it was a problem to setup in child.
Is it okay or you can advise me something?